recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - python >=3
     - nbconvert >=5.5.0,<6
-    - jupyter_server ==0.1.0,<0.2.0
+    - jupyter_server ==0.1.1,<0.2.0
```

Comment:
>=0.1.1 ?

Reply:
(the key here was that it was `==`, not that it was `0.1.0` :)

Reply:
Do you mean, I should change to `>=0.1.0`, or that I originally had started typing this, but I've already fixed/changed in 668a0dc  ?

Reply:
I mean that it should be:
```suggestion
    - jupyter_server =>0.1.0,<0.2.0
```